src/csharp/Grpc.IntegrationTesting/ClientRunners.cs

```diff
@@ -55,6 +58,15 @@ public class ClientRunners
     {
         static readonly ILogger Logger = GrpcEnvironment.Logger.ForType<ClientRunners>();
 
+        // Profilers to use for clients.
+        static readonly BlockingCollection<BasicProfiler> profilers = new BlockingCollection<BasicProfiler>();
+
+        internal static void AddProfiler(BasicProfiler profiler)
```

Comment:
Actually, I usually only want profiler for the first client thread (and I'd do profiling mostly for single-client scenario) so this serves the purpose.